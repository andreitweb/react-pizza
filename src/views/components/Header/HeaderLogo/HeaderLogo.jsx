import React from 'react';

import PizzaLogo from './../../../assets/img/pizza-logo.svg';

const HeaderLogo = () => {
	return (
		<div className="logo">
			<img width="38" src={PizzaLogo} alt="Pizza logo"/>
			<div>
				<h1>React Pizza</h1>
				<p>самая вкусная пицца во вселенной</p>
			</div>
		</div>
	);
}

export default HeaderLogo;