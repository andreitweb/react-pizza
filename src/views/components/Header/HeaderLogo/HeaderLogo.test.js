import React from 'react';
import renderer from 'react-test-renderer';

import HeaderLogo from './HeaderLogo';

it('should render with no crushing', function () {
	const component = renderer.create(<HeaderLogo/>)
	const tree = component.toJSON();
	expect(component).toMatchSnapshot();
});