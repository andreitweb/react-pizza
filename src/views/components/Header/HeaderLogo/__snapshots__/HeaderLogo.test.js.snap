// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with no crushing 1`] = `
<div
  className="logo"
>
  <img
    alt="Pizza logo"
    src="pizza-logo.svg"
    width="38"
  />
  <div>
    <h1>
      React Pizza
    </h1>
    <p>
      самая вкусная пицца во вселенной
    </p>
  </div>
</div>
`;
